src/style-spec/reference/v8.json

```diff
@@ -2033,255 +2033,570 @@
         "doc": "Gets the feature's id, if it has one.",
-        "group": "Feature data"
+        "group": "Feature data",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.41.0"
+          }
+        }
       },
       "zoom": {
         "doc": "Gets the current zoom level.  Note that in style layout and paint properties, [\"zoom\"] may only appear as the input to a top-level \"step\" or \"interpolate\" expression.",
-        "group": "Zoom"
+        "group": "Zoom",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.41.0"
+          }
+        }
       },
       "heatmap-density": {
         "doc": "Gets the kernel density estimation of a pixel in a heatmap layer, which is a relative measure of how many data points are crowded around a particular pixel. Can only be used in the `heatmap-color` property.",
-        "group": "Heatmap"
+        "group": "Heatmap",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.41.0"
+          }
+        }
       },
       "+": {
         "doc": "Returns the sum of the inputs.",
-        "group": "Math"
+        "group": "Math",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.41.0"
+          }
+        }
       },
       "*": {
         "doc": "Returns the product of the inputs.",
-        "group": "Math"
+        "group": "Math",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.41.0"
+          }
+        }
       },
       "-": {
         "doc": "For two inputs, returns the result of subtracting the second input from the first. For a single input, returns the result of subtracting it from 0.",
-        "group": "Math"
+        "group": "Math",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.41.0"
+          }
+        }
       },
       "/": {
         "doc": "Returns the result of floating point division of the first input by the second.",
-        "group": "Math"
+        "group": "Math",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.41.0"
+          }
+        }
       },
       "%": {
         "doc": "Returns the remainder after integer division of the first input by the second.",
-        "group": "Math"
+        "group": "Math",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.41.0"
+          }
+        }
       },
       "^": {
         "doc": "Returns the result of raising the first input to the power specified by the second.",
-        "group": "Math"
+        "group": "Math",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.41.0"
+          }
+        }
       },
       "sqrt": {
         "doc": "Returns the square root of the input.",
-        "group": "Math"
+        "group": "Math",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "0.42.0"
```

Comment:
> **0.42.0 (November 10, 2017)**
> - Add sqrt expression #5493

https://github.com/mapbox/mapbox-gl-js/blob/master/CHANGELOG.md#-features-and-improvements-3